src/pybind/mgr/dashboard/frontend/src/app/ceph/block/iscsi-target-details/iscsi-target-details.component.ts

```diff
@@ -75,26 +76,25 @@ export class IscsiTargetDetailsComponent implements OnChanges, OnInit {
 
   private generateTree() {
     this.metadata = { root: this.selectedItem.target_controls };
-
     const cssClasses = {
       target: {
-        expanded: 'fa fa-fw fa-bullseye fa-lg'
+        expanded: Icons.width + ' ' + Icons.large + ' ' + Icons.bullseye
```

Comment:
What about using this?
```suggestion
        expanded: _.join([Icons.width, Icons.large, Icons.bullseye], ' ')
```